### PR TITLE
ability to show custom error message for webview

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
Ref: #882 

Show custom error web view if file "assets/customError.html" exists.
Currently a dummy file exists for custom build, for regular build there is no such file and therefore the normal error message is shown.

### CODE REVIEW

**Comment by AndyScherzinger** [comment]
👍 code looks fine to to me, didn't test though since it is "modified only"

**Comment by aleister09** [comment]
It could be the same for WebLogin, if you want I can change it

Also It would be better load the WebError from RAW to have a WebError by each language, what do you think?

**Comment by mario** [comment]
Yes, we could/should support languages, and default to english if one isn't available.

Same for login.

Feel free to commit to PR and improve it.

**Comment by mario** [comment]
👍  looks fine, but @AndyScherzinger can test it :P

File: src/main/java/com/owncloud/android/authentication/AuthenticatorActivity.java [hunk 1]
```diff
@@ -336,6 +339,22 @@ public boolean shouldOverrideUrlLoading(WebView view, String url) {
...
+                try {
+                    while (( line = buffreader.readLine()) != null) {
+                        text.append(line);
+                        text.append('\n');
+                    }
+                } catch (IOException e) {
+                    Log_OC.e(TAG,e.getMessage());
+                    return;
+                }
+                mLoginWebView.loadData(text.toString(),"text/html; charset=UTF-8", null);
```
Comment (AndyScherzinger) [inline src/main/java/com/owncloud/android/authentication/AuthenticatorActivity.java]: @aleister09 @tobiasKaminsky @mario the ```onReceiveError(...)``` in both classes is basically the same code except the ```...loadData(...``` call which I think could be ```view.loadData(...``` which is a method input parameter. So I woudl have guessed this code could be centralized.

**Comment by AndyScherzinger** [comment]
Just one minor issue related to duplicated code :)

**Comment by aleister09** [comment]
I changed the duplicated code to Utils

**Comment by mario** [comment]
👍

**Comment by AndyScherzinger** [comment]
:+1:

**Comment by tobiasKaminsky** [comment]
Thank you for your help @aleister09 .
Just a minor enhancement, it is better to set the line wrap to 120 as this makes it much more easier to review changes/diffs here on github, otherwise you have to scroll horizontally.

**Pull request merged**